Silicon/ElkhartlakePkg/Include/Register/MrcOemPlatform.h

```diff
@@ -0,0 +1,47 @@
+/** @file
+  Sample implementation of SA_FUNCTION_CALLS member functions that are used by MRC,
+  and SPD read routines.

```

Comment:
Move the content into RtcRegs.h since it is not related with MRC?

Reply:
Moved to RtcRegs.h, thanks for the review.